ios/Sources/Component/SearchController.swift

```diff
@@ -5,29 +5,32 @@ public class SearchController: NSObject {
 
     public let wrappedValue = UISearchController()
     private let searchTextDidChangeTo: (String) -> Void
-    private let willDismissSearchController: () -> Void
+    private let isSearchTextEditing: (Bool) -> Void
 
     public init(
         searchBarPlaceHolder: String? = nil,
         searchTextDidChangeTo: @escaping (String) -> Void,
-        willDismissSearchController: @escaping () -> Void
+        isSearchTextEditing: @escaping (Bool) -> Void
     ) {
         self.searchTextDidChangeTo = searchTextDidChangeTo
-        self.willDismissSearchController = willDismissSearchController
+        self.isSearchTextEditing = isSearchTextEditing
         super.init()
         wrappedValue.searchBar.delegate = self
-        wrappedValue.delegate = self
+        // W/A first time showing search result, black blur view does not work correctly
+        wrappedValue.obscuresBackgroundDuringPresentation = false
```

Comment:
In current iOS development with UIKit, `obscuresBackgroundDuringPresentation` help us to show view like [this](https://www.figma.com/file/D2VLqh0xOXbH0zB6cTz053/DroidKaigi_2021_official_App-iOS?node-id=143%3A1492).
In this app, after any result(including empty) has found, black blur view should be hidden on the first time to search. This is why I choose to use `searchBarTextDidBeginEditing` to create custom view for this.